autoload/vebugger.vim

```diff
@@ -47,6 +47,8 @@ function! s:f_debugger.invokeReading() dict
     endfor
     for l:k in keys(l:newLines)
         for l:line in l:newLines[l:k]
+			" remove ANSI Colors
```

Comment:
This is a separate issue than the `language messages` one, and deserve a separate PR. With a better commit message.